safeweb/http.go

```diff
@@ -195,6 +196,20 @@ func NewServer(config Config) (*Server, error) {
 	return s, nil
 }
 
+// getMoreSpecificPattern returns either a or b, depending on which pattern
+// contains more pathname components (i.e. is more specific). If they are
+// equally specific, returns an empty string.
+func getMoreSpecificPattern(a, b string) string {
+	aLen := strings.Count(path.Clean(a), "/")
+	bLen := strings.Count(path.Clean(b), "/")
+	if aLen > bLen {
+		return a
+	} else if bLen > aLen {
```

Comment:
nit: just a tiny more idiomatic
```suggestion
	}
	if bLen > aLen {
```